### ISSUE TITLE
Sniper R rework

### ISSUE THREAD

**Issue by DoctorGester** [DoctorGester/crumbling-island-arena#288]
Requires a huge bunch of work in visuals department.

Channeling. Sniper activates a jetpack and starts using his weapon as a flamethrower. Slowly flies forward when the ability is active and can activate E to get a speed boost. Covers the ground in a cone in flames dealing periodic damage during the whole duration.

**Comment by 13lackHawk** [comment]
What is maximum damage for this ulti?

**Comment by Chiyukii** [comment]
Phoenix approves

**Comment by DoctorGester** [comment]
Yeah this is kind of similar to phoenix, except you can move and damage in a cone and you can have a speed boost etc. Total damage is not set but it can easily be 7-8.
